flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -300,7 +300,7 @@ function start_and_wait_for_tm {
 }
 
 function check_logs_for_errors {
-  if grep -rv "GroupCoordinatorNotAvailableException" $FLINK_DIR/log \
+  error_count=$(grep -rv "GroupCoordinatorNotAvailableException" $FLINK_DIR/log \
```

Comment:
Please add the description to the commit message why this change was needed. Maybe a bit extended description from the PR?

Btw, is this a hotfix and it doesn't have Jira ticket?